### DISCUSSION TITLE
pkg installer file for macOS

### DISCUSSION THREAD

**Discussion by PC-Support-Services** [rustdesk/rustdesk#9913] [General]
Can we get a proper pkg installer file for macOS?

**Comment by xlionjuan** [comment]
is this needed?

**Reply by PC-Support-Services** [reply]
At the moment the software has to be manually installed. With a pkg file it will be installed automatically.

**Comment by xlionjuan** [comment]
https://formulae.brew.sh/cask/rustdesk

**Reply by PC-Support-Services** [reply]
That won't work with custom clients. 

Installing through brew still need to be manually installed once opened.
